tests/autoyast/installation.pm

```diff
@@ -149,6 +151,8 @@ sub run {
     mouse_hide(1);
     check_screen \@needles, $check_time;
     until (match_has_tag('reboot-after-installation') || match_has_tag('bios-boot') || match_has_tag('autoyast-stage1-reboot-upcoming')) {
+        last if match_has_tag('linux-login-casp');
```

Comment:
Let's just put in the line above. Other than that let's think about more scalable solution, which we can introduce later.

Reply:
Done